superx/static/js/script.js

```diff
@@ -1,25 +1,45 @@
 // JavaScripts functions
 
+// Check if item is already in cart
+function addItemIfInCart(product_id, product_name){
+  //test 1- console.log("yes");
+  let b = false;
```

Comment:
do you know the difference between let and var? and are you purposefully using let?

Reply:
used the function in a different location which required let

recommited with to var